server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```

Comment:
same problem

Reply:
I removed the "<p>"s. Thanks for your suggestion.